### ISSUE TITLE
Any example using nats?

### ISSUE THREAD

**Issue by bbigras** [Rustixir/tokio_sky#1]

**Comment by Rustixir** [comment]
No, but its very easy just create a struct that implement producer or processor methods, and can create a factory with params to run it with run_topology ....

I will implement a example with nats Tomorrow 😉
